src/components/AdminAppointmentDetailsPopup.tsx

```diff
@@ -141,11 +141,12 @@ const AdminAppointmentDetailsPopup = ({
       const hh = Math.floor(startHourInMinute / 60); // getting hours of day in 0-24 format
 
       const mm = startHourInMinute % 60; // getting minutes of the hour in 0-55 format
+      const ampm = hh >= 12 ? 'PM' : 'AM';
+      const hh12 = hh === 0 ? 12 : hh > 12 ? hh - 12 : hh;
```

Comment:
Hey @NuriyaAkh,
Could you please simplify the nested conditional statement at Line # 145? It is not a good practice. Though it is succinct but hard to get the intent of the code at the first glance.